nb/projects/pentacular/lego_sorter/lego_sorter.md

```diff
@@ -3,8 +3,8 @@
 ```JavaScript
 const grid = Hexagon(16)
   .seq(
-    { from: -4, to: 4 },
-    { from: -4, to: 4 },
+    { from: -4, upto: 4 },
```

Comment:
This is more clear 👍 👍

Reply:
Yeah, it was annoying me for a long time. :)